arcjet/index.ts

```diff
@@ -523,6 +552,86 @@ export function slidingWindow<
   return rules;
 }
 
+export function sensitiveInfo<Custom extends string>(
+  options: SensitiveInfoOptions<Custom>,
+  ...additionalOptions: SensitiveInfoOptions<Custom>[]
+): Primitive<{}> {
+  const rules: ArcjetSensitiveInfoRule<{}, Custom>[] = [];
+
+  // Always create at least one EMAIL rule
+  for (const opt of [
+    options ?? ({} as SensitiveInfoOptions<Custom>),
+    ...additionalOptions,
+  ]) {
+    const mode = opt.mode === "LIVE" ? "LIVE" : "DRY_RUN";
+    // TODO: Filter invalid email types (or error??)
+
+    const redactOpts = {
+      allow: options?.allow || [],
+      deny: options?.deny || [],
+      contextWindowSize: opt.contextWindowSize || 1,
+      customDetect: options?.detect,
```

Comment:
```suggestion
      detect: options?.detect,
```
Can just be named `detect` here too, right?